tests/test_main.py

```diff
@@ -70,3 +75,14 @@ def test_detect_language_german_with_our_language_specification(self):
     def test_detect_language_german_with_module_language_specification(self):
         result = detect_language(self.texts["german"], LANGUAGES)
         self.assertEqual(result.lower(), 'german')
+
+    def test_stats(self):
+        """Should return percentages of words occurrences in text for each language"""
+        expected = {
+            'German': '51.55%',
+            'Spanish': '10.1%'
+        }
+        self.assertEqual(calculate_percentages(self.text_occurrences), expected)
```

Comment:
First time peer reviewing! Have nothing to add. Really cool how you included stats calculation!